prime-angular-frontend/src/main.ts

```diff
@@ -17,6 +17,7 @@ fetch('/assets/config-map.json')
     let appConfig = defaultAppConfig;
 
     if (configMap) {
+      // TODO mohKeyCloakConfig will eventually have configuration applied throughout the environments
```

Comment:
`mohKeyCloak` config will be added to OCP4 when the GIS team deploys out their test environment at the end of next week or early the following week.